postmarketOS/pmbootstrap
# Close #871: Enable binary repository

It should be basically working already. ~~Not all packages have been compiled, some got stuck while building and need further investigation~~. Binary repository is located here: http://postmarketos.brixit.nl

The builder does not automatically compile packages yet, I need to trigger it manually as of now. But that is planned (not in scope of this PR).

### How to test
* Check out the branch
* Whenever you try to build something, that exists in the binary repo, it should get skipped
* Whenever you install something, that exists in the binary repo, it should get taken from there
* See the packages here (almost everything, except firmware packages, tell me if I've missed anything): http://postmarketos.brixit.nl/

### TODO
- [x] update the README.html of the mirrors
- [x] ~~use HTTPS mirror if possible (see below, right now it uses HTTP)~~ nope
- [x] update "keys" testcase to pull Alpine's keys out of the `alpine-keys` package we install anyway when setting up the chroot (so we can enable the testcase in Travis, it is currently disabled as it downloads the whole aports git repo) and to add the postmarketOS key
- [x] check upgrade path from pmbootstrap without the mirror configured to this version
- [x] fix regression in config loading/saving for `timestamp_based_rebuild`
